uaclient/defaults.py

```diff
@@ -7,11 +7,11 @@
 
 DEFAULT_CONFIG_FILE = '/etc/ubuntu-advantage/uaclient.conf'
 BASE_AUTH_URL = 'https://login.ubuntu.com'
-BASE_SERVICE_URL = 'https://uaservice.canonical.com'
+BASE_SERVICE_URL = 'https://contracts.canonical.com'
```

Comment:
Should we rename this to `BASE_CONTRACT_URL`?

Reply:
yes; and done.

Reply:
done. good thought.